api/tests/test_application_api.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+
+import pytest
+from django.utils import timezone
+from rest_framework.reverse import reverse
+
+from applications.models import Application
+
+
+@pytest.mark.django_db
+def test_application_create(
+    purpose,
+    reservation_purpose,
+    organisation,
+    person,
+    application_period,
+    user_api_client,
+    user,
+):
+    assert Application.objects.count() == 0
+    data = {
```

Comment:
So the idea is that in the application creation process we first create separate organization, contact person etc and only after that we make another call to API to create application with these? 

So its possible that we have organisations and contact persons in db without application relation for them?

Reply:
I think organization probably should be separate table since it came up that it in the future might come from some external source etc. Contact person could probably be part of the application itself, doesn't look like it has much reusability, i can refactor that in separate merge request.